src/Classes/Factories/HiringPartnerModelFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: imogen
+ * Date: 27/11/2018
+ * Time: 15:45
+ */
+
+namespace Portal\Factories;
+
+use Portal\Models\HiringPartnerModel;
+
+class HiringPartnerModelFactory
+{
+    /**
+     * @param $c
+     * @return HiringPartnerModel
+     */
+    public function __invoke($c)
+    {
+        $db = $c->dbConnection;
```

Comment:
Should this not be get $c->get('dbConnection')?

Reply:
Not necessary, $c->x is the same as $c->get('x')